data_platform_helpers/tests_helpers/vms/ha_helpers.py

```diff
@@ -54,7 +56,107 @@ async def restore_network_for_unit_without_ip_change(machine_name: str) -> None:
     subprocess.check_call(limit_set_command.split())
 
 
-# TODO add these network helpers:
-# get_controller_machine
-# is_machine_reachable_from
-# wait_network_restore
+def is_machine_reachable_from(origin_machine: str, target_machine: str) -> bool:
```

Comment:
unrelated to this PR, but may be we can address it in this PR anyways

the following helpers are not async:
- cut_network_from_unit_without_ip_change
- restore_network_for_unit_with_ip_change
- restore_network_for_unit_without_ip_change